packages/react-components/src/alert/docs/Alert.stories.mdx

```diff
@@ -20,6 +20,17 @@ import { Heading, InnerHeading } from "@react-components/typography";
     githubPath="/packages/react-components/src/alert/src"
 />
 
+## Accessibility
+
+An alert comes with the `alertdialog` role and should be used to display a confirmation message, an alert, error, or warning.
+
+It requires:
+- The focus to be trapped inside the Alert.
```

Comment:
"the Alert" -> "the alert"

For consistency, everywhere we don't capitalize the components name